src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/designforextension/InputDesignForExtensionsLocalAnnotations.java

```diff
@@ -0,0 +1,12 @@
+package com.puppycrawl.tools.checkstyle.checks.design.designforextension;
+
+public class InputDesignForExtensionsLocalAnnotations
+{
+    public @interface Rule {
```

Comment:
All inputs should be compiled by itself only. The same as we demand from users on when they report issue.